userspace/libscap/scap_bpf.c

```diff
@@ -1487,3 +1493,52 @@ int32_t scap_bpf_get_n_tracepoint_hit(scap_t* handle, long* ret)
 
 	return SCAP_SUCCESS;
 }
+
+int32_t scap_bpf_set_simple_mode(scap_t* handle)
```

Comment:
While i was there, i added support for scap_enable_simpledriver_mode() for bpf probe!